includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,271 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $data = [
+		"logo" => [
+			"header" => [
+				"type" => "link-image",
+				"href" => "http://fandom.wikia.com",
+				"image" => "wds/full_fandom_logo",
+				"title" => [
+					"type" => "text",
+					"value" => "Fandom",
+				]
+			]
+		],
+		"verticals" => [
+			"links" => [
+				[
+					"type" => "link-branded",
+					"brand" => "tv",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-tv",
+					],
+					"href" => "http://tv.wikia.com"
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "games",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-games",
+					],
+					"href" => "http://games.wikia.com",
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "movies",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-movies",
+					],
+					"href" => "http://movies.wikia.com",
+				],
+			],
+		],
+		"wikis" => [
+			"header" => [
+				"type" => "link-branded",
+				"brand" => "wikis",
+				"href" => "#",
+				"title" => [
+					"type" => "translatable-text",
+					"key" => "global-navigation-brandlink-vertical-wikis",
+				],
+			],
+			"links" => [
+				[
+					"type" => "link-text",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-explorewikis",
+					],
+					"href" => "#",
+				],
+				[
+					"type" => "link-text",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-wikis-communitycentral",
+					],
+					"href" => "#",
+				],
+			],
+		],
+		"search" => [
+			"module" => [
+				"type" => "search",
+				"results" => [
+					"url" => "http://wikia.com/search",
+					"param" => "query",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "param" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)